From fd593a115f3042279fdc60f12917add9def5aa08 Mon Sep 17 00:00:00 2001
From: Vineet Gupta <vgupta@synopsys.com>
Date: Wed, 10 May 2017 11:55:46 -0700
Subject: [PATCH] Revert "Changed filename for the expected dynamic loader for
 glibc."

This reverts commit da44dae59150e754042f4bb8cb5bcfa1f15a82bb.

This is to pair with name in glibc "ld-linux.so.2" which is likely
to be the name accepted in glibc upstreaming !

Signed-off-by: Vineet Gupta <vgupta@synopsys.com>
---
 gcc/config/arc/linux.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/arc/linux.h b/gcc/config/arc/linux.h
index 725fa534a41b..06f936a48ac9 100644
--- a/gcc/config/arc/linux.h
+++ b/gcc/config/arc/linux.h
@@ -29,7 +29,7 @@ along with GCC; see the file COPYING3.  If not see
     }						\
   while (0)
 
-#define GLIBC_DYNAMIC_LINKER   "/lib/ld-linux-arc.so.1"
+#define GLIBC_DYNAMIC_LINKER   "/lib/ld-linux.so.2"
 #define UCLIBC_DYNAMIC_LINKER  "/lib/ld-uClibc.so.0"
 
 /* Note that the default is to link against dynamic libraries, if they are
-- 
2.7.4

